package.json

```diff
@@ -13,6 +13,17 @@
     "android": "react-native run-android",
     "snyk-protect": "snyk protect"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```

Comment:
Do you agree we should remove it?

Reply:
yes I agree that it should run before every commit, not just *js files